### PR TITLE
Fix bug in api/copr-builds

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: packit_service/service/api/copr_builds.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def get(self, id):
             if project:
                 build_dict["repo_namespace"] = project.namespace
                 build_dict["repo_name"] = project.repo_name
-                build_dict["pr_id"] = build.trigger.get_trigger_object().pr_id
+                build_dict["pr_id"] = build.job_trigger.trigger_id
```
Comment (lachmanfrantisek) [inline packit_service/service/api/copr_builds.py:124]: This does not look like `pr_id` -- it's a database `id` of the specific trigger object (like `GitBranchModel`, `PullRequestModel`).

If you expect `build` for pull-request -- so the trigger object is `PullRequestModel`, you can use this:

```suggestion
                build_dict["pr_id"] = build.job_trigger.get_trigger_object().pr_id
```

But you need to check, that you get `PullRequestModel` and not e.g. `GitBranchModel`.

---

`job_trigger` is only a mapper between `CoprBuildModel` and specific trigger like `PullRequestModel`, `GitBranchModel`, `ProjectReleaseModel`, ...

File: packit_service/models.py [hunk 2]
```diff
@@ -390,6 +390,12 @@ def get_project(self) -> Optional[GitProjectModel]:
             return None
         return trigger_object.project
 
+    def get_pr_id(self) -> Optional[int]:
+        trigger_object = self.job_trigger.get_trigger_object()
+        if isinstance(trigger_object, PullRequestModel):
+            return trigger_object.pr_id
+        return None
```
Comment (lachmanfrantisek) [inline packit_service/models.py:397]: :+1:

File: tests_requre/database/test_models.py [hunk 3]
```diff
@@ -95,6 +95,7 @@ def test_create_copr_build(clean_before_and_after, a_copr_build_for_pr):
     # we will check if a_copr_build has build_submitted_time value that's within the past hour
     time_last_hour = datetime.utcnow() - timedelta(hours=1)
     assert a_copr_build_for_pr.build_submitted_time > time_last_hour
+    assert a_copr_build_for_pr.get_pr_id() == 342
```
Comment (lachmanfrantisek) [inline tests_requre/database/test_models.py:98]: Maybe also some `None` scenario for other triggers?

Comment (lbarcziova) [reply tests_requre/database/test_models.py]: added

**Comment by TomasTomecek** [comment]
(I actually wanted a test for the API itself - never mind, will ask in the next PR related to API :D )

**Comment by csomh** [comment]
What was the bug about?

**Comment by lbarcziova** [comment]
Getting `pr_id` value was called wrongly

**Pull request merged**